OpenTrans.net/OrderWriter.cs

```diff
@@ -97,6 +101,7 @@ public void Save(Order order, Stream stream)
             Writer.WriteEndElement(); // !ORDER_ITEM_LIST
 
             Writer.WriteStartElement("ORDER_SUMMARY");
+            _writeOptionalElementAmount(Writer, "TOTAL_AMOUNT ", order.OrderSummary.TotalAmount);
```

Comment:
Is it possible to automatically calculate TotalAmount? Imho that'd make much more sense than adding this as a field.

Reply:
Specification quote:

> Gross price including surcharges, deductions and all taxes. Where no price per item can be given as the item level (e.g. bills of materials where explosion is not possible), the total price can be entered here.

This is actually not quite clear to me.
It seems like the total amount may differ from sum(items).
The element is optional and always calculating it, we must provide a possibility to omit it and to override calculated value?

Reply:
after short look into the specs, it seems that all amount are optional, so it's not possible to calculate it by default.
Am i missing something?

Reply:
Another argument would be:
Its better to keep logic away from format converters.
So since afaik there is no mandatory function for this calculation defined in the specs, i propose to give the control for calculation to the applications business domain.

Reply:
You are probably right. Let's leave it like it is.